packages/components/src/TitleBlock/_docs/TitleBlock--api-specification.mdx

```diff
@@ -1,23 +1,43 @@
-import { Canvas, Meta } from '@storybook/blocks'
+import { Canvas, Controls, Meta } from '@storybook/blocks'
 import { ResourceLinks, KAIOInstallation, RenameNotice } from '~storybook/components'
-import * as TitleBlockZenStories from './TitleBlockZen.stories'
+import * as TitleBlockStories from './TitleBlock.stories'
 
-<Meta of={TitleBlockZenStories} />
+<Meta title="Components/TitleBlock/API Specification" />
 
-# TitleBlockZen
+# TitleBlock
 
 <ResourceLinks
-  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/TitleBlockZen"
-  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3075605782/Title+Block"
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/TitleBlock"
+  designGuidelines="/?path=/docs/components-titleblock-usage-guidelines--docs"
 />
 
-<RenameNotice />
-
-<KAIOInstallation exportNames="TitleBlockZen" />
+<KAIOInstallation exportNames="TitleBlock" />
 
 ## Overview
 
-<Canvas of={TitleBlockZenStories.Playground} />
+`TitleBlock` A consistent pattern to allow users to navigate and complete tasks easily.
+
+<Canvas of={TitleBlockStories.Playground} />
+
+## Props
+
+<Controls
+  of={TitleBlockStories.Playground}
+  include={[
+    'avatar',
+    'breadcrumb',
+    'defaultAction',
+    'handleHamburgerClick',
+    'navigationTabs',
+    'primaryAction',
+    'secondaryActions',
+    'secondaryOverflowMenuItems',
+    'surveyStatus',
+    'title',
+  ]}
+/>
+
+## Options
 
 ## primaryAction
```

Comment:
Could you change any nested title under this `## Options` to an H3, i3 `### primaryAction`. Same for the Usage guidelines :)